### ISSUE TITLE
[CSS] highlighted known font-family part of unknown

### ISSUE THREAD

**Issue by stevenvachon** [sublimehq/Packages#751]
```css
:root {
  --body-font: Raleway, Arial, Helvetica Neue, Helvetica, sans-serif;
}
```

produces:

![screenshot](https://cloud.githubusercontent.com/assets/170197/21429427/aacfcb96-c82c-11e6-8b03-6a5955212144.png)

It probably shouldn't highlight "Helvetica" in "Helvetica Neue".

**Comment by MattDMo** [comment]
I just submitted PR #771 that will hopefully address this.

**Comment by stevenvachon** [comment]
Thanks!

**Comment by wbond** [comment]
This was fixed with 3889492cac950cb90b98db53d94ef47a6b0bd5fe
